bundles/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/SpeedTestConfiguration.java

```diff
@@ -20,7 +20,16 @@
  */
 public class SpeedTestConfiguration {
     public Integer refreshInterval = 20;
+    public Integer initialDelayMinutes = 5;
```

Comment:
```suggestion
    public Integer initialDelay = 5;
```

Adjust to documentation. No need to name it `Minutes`.